packages/legend-query/src/stores/QueryBuilderLambdaBuilder.ts

```diff
@@ -98,7 +99,12 @@ export const buildLambdaFunction = (
   );
 
   // build getAll()
-  const getAllFunction = buildGetAllFunction(_class, multiplicityOne);
+  let getAllFunction = buildGetAllFunction(_class, multiplicityOne);
+  if (_class instanceof Class && _class.stereotypes.length !== 0) {
+    getAllFunction = guaranteeNonNullable(
+      queryBuilderState.getAllFunctionState,
```

Comment:
See other comment. We should add to the parameters, not replace it of the getAllFunction if the property `versionPropertyParameter` has been set.